index.html

```diff
@@ -1020,7 +1022,8 @@ <h1 class="large-h1 text-white">Subscribe to stay informed</h1>
 							</ul>
 						</div>
 					</div>
-
+					<center>
```

Comment:
Please do not use `center` tag instead implement it using CSS.

Reply:
@geekyd can you tell me if its right now?
![screenshot_20171025_232617](https://user-images.githubusercontent.com/30254676/32014583-172ac3ee-b9dc-11e7-8843-083be1d8283d.png)

Reply:
Screen size is 1920*1080.